### PR TITLE
Clean up .gitignore

### LINKED ISSUE #558: Clean up and comment .gitignore
I think some of the files in [.gitignore](https://github.com/Courseography/courseography/blob/8f7c29b3b11ffc7fce93d97c2379d1dde73afbf3/.gitignore) right now are a bit outdated (i.e. aren't generated / don't need to be ignored anymore)

### PR DESCRIPTION
**Pull request by elliottsj**
Resolves #558

### CODE REVIEW

**Comment by christinem** [comment]
Also, when you click on "Post To Facebook", an SVG file is generated of whatever you were trying to post and saved locally in the hs directory. I wonder if hs/*.svg should be added as well, or just not have the generated svg files stored locally.

**Comment by Ian-Stewart-Binks** [comment]
@christinem Hmm. The image generation code should clean up after itself... I'll take a look at this.

File: .gitignore [hunk 1]
```diff
@@ -1,20 +1,19 @@
...
+# generated build files
+/hs/dist/
+
+# generated CSS
+/public/style/
+
+# generated resources
+/public/res/graphs/
+/public/res/courses/
+/public/res/courses-copy/
```
Comment (Ian-Stewart-Binks) [inline .gitignore]: I don't think that the last two entries are being used anymore.

File: .gitignore [hunk 2]
```diff
@@ -1,20 +1,17 @@
...
+*.sqlite3-wal
+
+# generated build files
+/hs/dist/
+
+# generated CSS
+/public/style/
+
+# generated resources
+/public/res/graphs/
```
Comment (david-yz-liu) [inline .gitignore]: We need to keep this for now. The graphs made manually (in yEd and Inkscape) are currently also stored in this directory. We can move them to a separate location.

**Comment by elliottsj** [comment]
I've updated the `.gitignore` to ignore just 1.svg and 2.svg: https://github.com/elliottsj/courseography/commit/fb04eb85d73a156e350a5b871f6cdee28a2cd35d

**Comment by elliottsj** [comment]
I've updated the `.gitignore` to ignore just 1.svg and 2.svg: https://github.com/elliottsj/courseography/commit/fb04eb85d73a156e350a5b871f6cdee28a2cd35d

**Pull request merged**